website/content/docs/connect/gateways/api-gateway/configuration/gatewaypolicy.mdx

```diff
@@ -0,0 +1,257 @@
+
+Specifies the version of the Consul API for integrating with Kubernetes. The value must be `consul.hashicorp.com/v1alpha1`.
+
+#### Values
+
+- Default: None
+- This field is required.
+- String value that must be set to `consul.hashicorp.com/v1alpha1`.
+
+### `kind`
+
+Specifies the type of configuration entry to implement. Must be set to `GatewayPolicy`.
+
+#### Values
+
+- Default: None
+- This field is required.
+- Data type: String value that must be set to `GatewayPolicy`.
+
+### `metadata`
+
+Map that contains an arbitrary name for the resource and the namespace it applies to.
+
+#### Values
+
+- Default: None
+- Data type: Map
+
+### `metadata.name`
+
+Specifies a name for the resource. The name is metadata that you can use to reference the resource when performing Consul operations, such as applying the resource to a specific cluster. 
+
+#### Values
+
+- Default: None
+- This field is required.
+- Data type: String
+
+### `metadata.namespace` 
+
+Specifies the namespace that the configuration applies to. Refer to [namespaces](/consul/docs/enterprise/namespaces) for more information.
+
+#### Values
+
+- Default: None
+- Data type: String
+
+### `spec`
+
+Map that contains the details about the gateway policy. The `apiVersion`, `kind`, and `metadata` fields are siblings of the `spec` field. All other configurations are children.
+
+#### Values
+
+- Default: None
+- This field is required.
+- Data type: Map
+
+### `targetRef`
+
+Map that contains references to the gateway that the policy applies to. 
+
+#### Values
+
+- Default: None
+- This field is required.
+- Data type: Map
+
+The following table describes the members of the `targetRef` map:
+
+| Parameter | Description | Data type | Default |
+| ---       | ---         | ---       | ---     |
+| `name` | Specifies the name of the API gateway that the policy attaches to. | String | None |
+| `kind` | Specifies the type of resource that the policy attaches to. Must be set to `Gateway`. | String | None |
+| `group` | Specifies the resource group. Must be set to `gateway.networking.kuberenetes.io`. | String | None |
+| `sectionName` | Specifies a part of the gateway that the policy applies to. | String | None |
+
+### `spec.override`
+
+Map that contains configurations to apply to listeners when the policy is attached to the gateway. The override settings have precedence over the [`spec.default` configurations](#spec-default).
+
+#### Values
+
+- Default: None
+- Data type: Map
+
+### `spec.override.jwt`
+
+Map that contains JWT verification configurations to apply to listeners when the policy is attached to the gateway. The override settings have precedence over the [`default` configurations](#listeners-default) as well as any route-specific JWT configurations defined in [`HTTPRouteAuthFilter` configurations](/consul/docs/connect/gateways/api-gateway/configuration/httprouteauthfilter).
+
+#### Values
+
+- Default: None
+- Data type: Map
+
+### `spec.override.jwt.providers`
```

Comment:
should this specify that it needs to reference an existing JWT provider config entry?

Reply:
I think we cover that in the usage part of the documentation. I'll add a link to the places you've called out.